compiler/src/java_plugin/cpp/java_generator.cpp

```diff
@@ -776,7 +776,7 @@ void GenerateService(const ServiceDescriptor* service,
   vars["MethodType"] = "io.grpc.MethodDescriptor.MethodType";
   vars["ServerMethodDefinition"] =
       "io.grpc.ServerMethodDefinition";
-  vars["BindableService"] = "io.grpc.stub.BindableService";
```

Comment:
@carl-mastrangelo Yes, it belongs to stubs but as @ejona86 noted, it has to be in `io.grpc` because we're using it in `ServerBuilder` and `ServerInterceptors`.

Feels a bit weird to me too, but seems like the only way to go (?).